chsdi/static/images/legends/ch.bafu.bundesinventare-trockenwiesen_flachmoore_de.png

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding="UTF-8" standalone="no" ?>
+<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/ogc http://schemas.opengis.net/wms/1.3.0/exceptions_1_3_0.xsd">
+<ServiceException code="LayerNotDefined">
+msWMSGetLegendGraphic(): WMS server error. Invalid layer given in the LAYER parameter. A layer might be disabled for this request. Check wms/ows_enable_request settings.
+</ServiceException>
+</ServiceExceptionReport>
```

Comment:
I don't think this is want you wanted to have. I may add a check in download legend utility to avoid these cases.

Reply:
I think I indicated the wrong layer name when I built the legends... Like ` make legend ch.bafu.bundesinventare-trockenwiesen_flachmoore_de` instead of  ` make legend ch.bafu.bundesinventare-flachmoore_de`. I will remove them right away.
You're right a check would be nice. Just to be sure, you want me to add it in this script `scripts/downloadlegends.sh`?

Reply:
yep, would be great! Thanks!